pkgs/os-specific/linux/util-linux/default.nix

```
@@ -1,5 +1,7 @@
 { lib, stdenv, fetchurl, pkg-config, zlib, shadow, libcap_ng
-, ncurses ? null, pam, systemd ? null
+, ncurses ? null, pam
+, systemd
+, systemdSupport ? systemd!=null && stdenv.isLinux
```

```suggestion
, systemdSupport ? stdenv.isLinux
```
systemd is only null if the end user overwrites it to that and overwriting packages to null breaks things. It is fine to remove this and the only valid use right now it bootstrapping since we don't have all packages there. I don't think this package is used for bootstrapping so we can also remove the ncurses null shenanigans.

---

Unfortunately `all-packages.nix` overwrites it to `null`.

I will fix that too.